### ISSUE TITLE
Add the option to configure the colour of code font

### ISSUE THREAD

**Issue by VictorVow** [ryanpcmcquen/workflowyCodeFormatter#22]
For instance Dynalist uses a red colour for code blocks which makes it far more readable. I'd like to be able to use the same, or maybe try other readable colours.
![](https://i.imgur.com/m8QP58s.png)

For contrast this extension is hard to read when using the dark theme:
![](https://i.imgur.com/3wHYmwi.png)

**Comment by ryanpcmcquen** [comment]
Yeah, this is possible but I don't have time to implement it. Best workaround right now is a custom stylesheet.
